app/config.yml.erb

```diff
@@ -0,0 +1,14 @@
+:base_path: <%= "#{ENV['GEMSTASH_HOME']}/data" %>
+:cache_type: memory
+:rubygems_url: https://rubygems.org
+:puma_threads: 16
+:bind: tcp://0.0.0.0:9292
+:protected_fetch: false
+:fetch_timeout: 20
+:log_file: :stdout
+
+# MySQL Adapter Settings
+<% if ENV['GEMSTASH_DB_ADAPTER'] == 'mysql2' %>
+:db_adapter: <%= ENV['GEMSTASH_DB_ADAPTER'] %>
+:db_url: <%= "#{ENV['GEMSTASH_DB_ADAPTER']}://#{ENV['GEMSTASH_DB_USERNAME']}:#{ENV['GEMSTASH_DB_PASSWORD']}@#{ENV['GEMSTASH_DB_HOST']}:#{ENV['GEMSTASH_DB_PORT']}/#{ENV['GEMSTASH_DB_DATABASE']}" %>
```

Comment:
It doesn't seem necessary to use `<%= ENV['GEMSTASH_DB_ADAPTER'] %>` [here](https://github.com/Intellection/docker-gemstash/blob/72caa7c1b70922eda578e6a7b7488478a2808d72/app/config.yml.erb#L12-L13), could we just set it to `mysql2`?